Templates/Common-Backend-Scripts/pipelineBackend.config

```diff
@@ -326,6 +330,22 @@ wdDeployPackageDir() {
 ## End of wazideploy-generate.sh, wazideploy-deploy.sh and wazideploy-evidences.sh parameters    ####
 #####################################################################################################
 
+#####################################################################################################
+## generateCleanupCommands.sh parameters ############################################################
+#####################################################################################################
+  
+# DBB Metadatastore Options String to configure Db2 or File Metadatastore in cleanup commands
+# e.g.
+# for file metadatastore
+#  cleanupDbbMetadataStoreOptions="--type file --location /u/github/"
+# for db2 metadatastore, reusing build settings
+#  cleanupDbbMetadataStoreOptions="--type db2 --user ${dbbMetadataStoreJdbcId} --password-file ${dbbMetadataStoreJdbcPwdFile} --db2-config ${dbbMetadataStoreJdbcConfigFile} --url ${dbbMetadataStoreJdbcUrl}"
+cleanupDbbMetadataStoreOptions=""
```

Comment:
We should not introduce a new variable for this cleanup, and compute the string in the `generateCleanupCommands.sh` based on the information that already present in the pipelineBackend.config file (dbbMetadataStoreJdbcId, dbbMetadataStoreJdbcPwdFile, dbbMetadataStoreJdbcConfigFile and dbbMetadataStoreJdbcUrl).

By the way, there should be a dbbMetadataStoreType (file|db2) parm and a dbbMetadataStoreFileLocation parm as well in pipelineBackend.config, to help compute the right connection string.

Reply:
Hi @M-DLB , 

> By the way, there should be a dbbMetadataStoreType (file|db2) parm and a dbbMetadataStoreFileLocation parm as well in pipelineBackend.config, to help compute the right connection string.

I understand the point here - however, I don't want to maintain two variables with the same purpose - one in the build framework and another time in the pipelineBackend.config maintaining for maintaining the dbb metadatastore type.

Could we leave it as is now? Users need to customise the pipelineBackend.config anyway, and there are clear samples provided.

Reply:
You already have variables for metadatstore duplicated in the build framework and in the pipelineBackend.config file. I think it would be cleaner to add the two that are missing (dbbMetadataStoreType (file|db2) parm and a dbbMetadataStoreFileLocation parm) to completeness, remove the `cleanupDbbMetadataStoreOptions` parm and compute it instead.
You should aim for consistency, instead of doing one way with DB2 and another way with file metadatastore.